src/setup-beam.js

```diff
@@ -300,7 +309,7 @@ async function getElixirVersions(hexMirrors) {
     .forEach((line) => {
       const elixirMatch =
         line.match(/^v?(.+)-otp-([^ ]+)/) || line.match(/^v?([^ ]+)/)
-      const elixirVersion = maybePrependWithV(elixirMatch[1])
```

Comment:
This is no longer required, since internally all versions will be stripped of `v` and it'll only be added when required for external requests.